.ci/oci-o11y-deplyment.sh

```diff
@@ -0,0 +1,8 @@
+# exit immediately when a command fails
+set -e
+# only exit with zero if all commands of the pipeline exit successfully
+set -o pipefail
+# error on unset variables
+set -u
+
+echo "Hello World"
```

Comment:
please add a main function, you can see an example here https://github.com/redhat-appstudio/o11y/pull/2/files#diff-ce25e369a9df9edc525425cc144fc8c40599fff3cc1e5058b64336d71af67162